doc/quickstart/find_data.rst

```diff
@@ -131,6 +140,74 @@ ERA5
   of both liquid and solid phases to vapor (from underlying surface and vegetation)."
   Therefore, the ERA5 (and ERA5-Land) CMORizer switches the signs of ``evspsbl`` and ``evspsblpot`` to be compatible with the CMOR standard used e.g. by the CMIP models.
 
+.. _read_native_era5_grib:
+
+ERA5 (in GRIB format available on DKRZ's Levante)
```

Comment:
Mention that ERA5 data in grib format can also be downloaded from ECMWF?

Reply:
I am not sure how useful that is in practice to be honest. All the facets and paths here are tailored towards the files on Levante. I *think* you'll get different paths if you download them manually since the names of the facets seem to be [different](https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-complete?tab=form) in the docs.

To me, it's easier for the user if we just recommend manual download of the files in netCDF format via era5cli.

Reply:
I'm not sure if we should recommend using the NetCDF data because 1) we have encountered issues with it in the past and 2) the NetCDF option is still listed as experimental and not suitable for production use on the CDS download page: https://cds.climate.copernicus.eu/datasets/reanalysis-era5-pressure-levels?tab=download.

Reply:
Ok, I am happy to include this in the documentation, but please be aware that I didn't test this and it would probably need some effort from the user side to make it work (either by putting the files in the correct directories or by setting up a new DRS).

Reply:
Added in https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/e4d642a465417cf17fca7d329150389f37b0f616

Reply:
Thanks!